talentsearch/resources/js/talentSearchConstants.ts

```diff
@@ -1,2 +1 @@
-export const BASE_URL = process.env.TALENTSEARCH_APP_URL as string;
-export const BASE_URL_DIR = process.env.TALENTSEARCH_APP_DIR as string;
+export const TALENTSEARCH_APP_DIR = process.env.TALENTSEARCH_APP_DIR as string;
```

Comment:
I am getting an ESLint error for this line since there is only a single `const` now.
Prefer default export: [import/prefer-default-export](https://github.com/import-js/eslint-plugin-import/blob/v2.25.3/docs/rules/prefer-default-export.md).